run.py

```diff
@@ -106,6 +126,19 @@ def config_validator():
         app.config['TESTING'] = True
         print("Starting server in ~*TEST MODE*~")
         app.run(host='localhost')
+    elif args.backup:
+        try:
+            command = ['ls', '-l', 'issues.db']
+            if execute_command(command)[1] == 0:
+                execute_command(['mkdir', '-p', app.config['BACKUP_DEFAULT_DEST']])
```

Comment:
Same thing here, using the `os` module. 

https://docs.python.org/2/library/os.html#os.mkdir

(And for mv as well below)